src/modules/integrator-guide/pages/corteza-studio/index.adoc

```diff
@@ -0,0 +1,103 @@
+include::ROOT:partial$variables.adoc[]
+
+= {PRODUCT_NAME} Studio
+:description: {PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences.
+
+{PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences. 
+
+[IMPORTANT]
+====
+For {PRODUCT_NAME} Studio, to function properly, you need to xref:devops-guide:installing-sass/index.adoc[Install dart-sass binaries] in your system and 
+`HTTP_WEBAPP_ENABLED` enviroment variable set to `true`.
+====
+
+== Customizing {PRODUCT_NAME} Web Applications
+
+There are two ways of customizing your {PRODUCT_NAME} web applications style:
+
+=== Branding
+
+In the admin area, navigate to the menu:User Interface[Settings] branding section.
+
+[annotation,role="data-zoomable"]
```

Comment:
Better!
But let's have a different take on this:

For cases where you want to point out things like new sections, I like to see some context on the image. A list of color blobs completely out of context could throw people off. So prefer to use things like annotations (boxes) or a blur.
If you're pointing out something small, use a box; if you're pointing out something larger, use a blur.

So for your case, I'd prefer to see something like this:
![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/1bc2cf29-2ab0-4a3c-9cfb-80b417bef493)

## How I got to this

**Important**: there is an extra thicker outline on the edge of some screenshots. I have an outline around the active window so ignore that :see_no_evil: 
![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/185f984a-512d-404d-87f4-74bc8e3bd05b)


1. Draw the red box tightly around the section you're pointing out

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/ab31c89a-4548-42b0-aad7-0021b969b7f3)

2. **Only when blurring**: go a bit into the section. In the styles editor, highlight the pixels, hold shift, and press one of the arrow keys. In the end it should look like this

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/b60686a7-62f9-4f66-a58e-df558afd91e4)
![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/dc255119-a08b-4700-baff-4ec07179a485)